src/Appwrite/Auth/Auth.php

```diff
@@ -251,9 +252,9 @@ public static function sessionVerify(array $sessions, string $secret)
                 $session->isSet('expire') &&
                 $session->isSet('provider') &&
                 $session->getAttribute('secret') === self::hash($secret) &&
-                $session->getAttribute('expire') >= \time()
+                $session->getAttribute('expire') >= DateTime::now()
             ) {
-                return (string)$session->getId();
+                return $session->getId();
```

Comment:
If we remove it here and its unnecessary, we can remove it from other places as well.

Reply:
I did not see more cases of this casting to string